docs/architecture/BACKEND_MODULES.md

```
@@ -0,0 +1,439 @@
+        Focus[FocusTimeModule]
+        Task[TaskModule]
+        Github[GithubModule]
+        Pet[UserPetModule]
+        Point[PointModule]
+    end
+
+    App --> Config
+    App --> DB
+    App --> AuthM
+    App --> Player
+    App --> Room
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)